pilot/cmd/pilot-agent/status/ready/probe_test.go

```diff
@@ -23,14 +23,15 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var probe = Probe{AdminPort: 1234}
+var probe Probe
```

Comment:
nit: does this need to be top level? Can you just do `probe := ...` in each test?